rtgui/icmpanel.cc

```diff
@@ -1079,22 +1067,22 @@ void ICMPanel::write(ProcParams* pp, ParamsEdited* pedited)
     }
 
     auto workingTRCsActiveRow = wTRC->get_active_row_number();
```

Comment:
If you compare to a signed type later, this should also be a signed type and not `auto`.